src/aro/Parser.zig

```diff
@@ -8679,6 +8687,9 @@ fn unExpr(p: *Parser) Error!?Result {
                 try p.err(tok, .not_assignable, .{});
                 return error.ParsingFailed;
             }
+            if (operand.qt.get(p.comp, .pointer)) |pointer| {
+                try p.checkPtrArithmeticAllowed(pointer, p.tok_i, operand.node);
```

Comment:
`get` doesn't handle `.invalid`, instead move this to `checkPtrArithmeticAllowed` behind a `.isInvalid()` check.